pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Topic.java

```diff
@@ -105,6 +105,10 @@ default void setProperty(String propertyName, Object value) {
         default Object getProperty(String propertyName) {
             return null;
         }
+
+        default boolean isChunked() {
```

Comment:
Do we need to add the override annotation in the implementation class like `MessagePublishContext`? I think it is good to have.